src/test/kotlin/com/cognifide/gradle/aem/test/InstallResponseBuilderTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.cognifide.gradle.aem.test
+
+import com.cognifide.gradle.aem.pkg.deploy.CriticalInstallationError
+import com.cognifide.gradle.aem.pkg.deploy.InstallResponse
+import com.cognifide.gradle.aem.pkg.deploy.InstallResponseBuilder
+import org.junit.Assert.assertTrue
+import org.apache.commons.io.IOUtils
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import java.io.File
+import java.io.FileInputStream
+
+
+@RunWith(Parameterized::class)
+class InstallResponseBuilderTest(filename: String, private val expectedError: CriticalInstallationError?) {
```

Comment:
generally speaking this test looks ugly :)

I am not sure about value of that test. what is more, now junit5 must be used and I am planning to migrate these few tests what we have which will be very easy.

I dislike very much `@Parameterized.Parameters()` / junit5 offers sth better.

to sum up, consider upgrading tests to junit5; try to refactor that test / reduce logic / complexity

Reply:
I just noticed that you are trying to commit 15 mb or more for testing data. OK,  fine you covered the bug what you fixed but I think the test is just to heavy for VCS etc.